app/src/main/AndroidManifest.xml

```diff
@@ -63,22 +64,15 @@
             -->
         <activity
             android:name=".clothingFront"
-
             android:exported="true" />
-        <!--
- duplicate login activity
-        <activity
-            android:name=".ui.login.LoginActivity"
-            android:exported="false"
-            android:label="@string/title_activity_login" />
-        -->
+
         <activity
             android:name=".closetPage.SavedOutfit"
             android:configChanges="orientation|keyboardHidden|screenSize"
 
             android:exported="true">
-        </activity>
 
+        </activity>
         <activity
             android:name=".clothingUpload"
             android:exported="false" />
```

Comment:
Should be exported to be able to run on the live emulator

Reply:
got it!! changed